lago/providers/libvirt/vm.py

```diff
@@ -488,11 +492,11 @@ def _get_qemu_kvm_path(self):
 
         return qemu_kvm_path
 
-    def _load_xml(self):
+    def _load_xml(self, libvirt_ver):
```

Comment:
I don't really get the point of this change since it does not make you cache libviryt_ver longer - it would probably be better to have a lazy loading self.libvirt_ver property.

Reply:
Done.